### PR TITLE
Fix stat var search bar dropdown in stat var explorer and download tool

### PR DESCRIPTION
**Pull request by juliawu**
Fixes a bug in our stat var explorer and download tool where the SV search results dropdown covers the search bar itself.

Also adjusts the spacing on the "X" in the search bar when typing to be more vertically centered.

Before:
Screenshot 2024-06-07 at 5 26 05 PM

After:
Screenshot 2024-06-07 at 5 24 37 PM
Screenshot 2024-06-07 at 5 24 17 PM
Screenshot 2024-06-07 at 5 25 43 PM

### CODE REVIEW

**Review by dwnoble — COMMENTED** [review]
Nice! Can you also make the dropdown's width match the width of the input container?

File: server/webdriver/tests/browser_test.py [hunk 1]
```diff
@@ -255,7 +255,8 @@ def test_stat_var_hierarchy(self):
     WebDriverWait(self.driver, self.TIMEOUT_SEC).until(loading_finished)
     first_result = self.driver.find_element(
         By.XPATH,
-        '//*[@id="stat-var-hierarchy-section"]/div[1]/div[1]/div[2]/div/div[1]')
+        '//*[@id="stat-var-hierarchy-section"]/div[1]/div[1]/div[1]/div/div[2]/div[1]'
```
Comment (dwnoble) [inline server/webdriver/tests/browser_test.py:258]: Possible refactor: `'//*[@id="stat-var-hierarchy-section"]//div[contains(@class, "statvar-hierarchy-search-results")]'`

Comment (juliawu) [reply server/webdriver/tests/browser_test.py]: Thanks for the tip! Refactored.

**Comment by juliawu** [comment]
> Nice! Can you also make the dropdown's width match the width of the input container?

Updated to full width. See screenshot: 
Screenshot 2024-06-11 at 12 20 13 PM

**Review by dwnoble — APPROVED** [review]
Thanks Julia!

**Pull request merged**